src/prefect/engine/task_runner.py

```diff
@@ -84,11 +84,11 @@ def __init__(
         self,
         task: Task,
         state_handlers: Iterable[Callable] = None,
-        result_handler: "ResultHandler" = None,
+        result_handler: "ResultHandler" = None, # todo: here
```

Comment:
I think we should remove this kwarg from runner classes, it doesn't serve much purpose AFAIK